src/plugins/jsonmanager/jsonmanager-doc-en.hbs

```diff
@@ -11,209 +11,182 @@
 ---
 <div class="wb-prettify all-pre"></div>
 <section>
```

Comment:
Global comment for this file (jsonmanager-doc-en.hbs)

What has changed? this file look the same as the original.